hikari/api/special_endpoints.py

```diff
@@ -1084,6 +1104,28 @@ def description(self) -> str:
             The description to set for this command.
         """
 
+    @property
+    @abc.abstractmethod
+    def description_localizations(self) -> undefined.UndefinedOr[typing.Mapping[str, str]]:
+        """Return the description locales set for this command."""
```

Comment:
```suggestion
        """The description locales set for this command."""
```

Reply:
Removing "Return" causes flake8 to fail.

Reply:
In that case you will have to find the correct wording for it.

I believe this should work:
```suggestion
        """Description locales set for this command."""
```

Reply:
It doesn't. Why is this change required?

Reply:
Because it is not a callable, it does not return anything. You just access it (essentially another attribute)